.prettierrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "singleQuote": true,
```

Comment:
Do we want to stick with single-quotes? Or take this opportunity to migrate to double-quotes (in the interest of keeping our prettier customization minimal)?

Reply:
The fact that on most keyboards a double quote requires using the shift key leads me to vote against this <3
This is one of the few customisations that the Prettier project allows for - which means that it must be broadly accepted as a reasonable option...

Reply:
I'm a single-quote kinda guy, so I'm easily convinced. I always just imagined that native English speakers preferred double quotes because that way strings with common English constructs like "I'm", "You're", "They've" etc require no escaping the way they do when single-qoutes are used for strings.

If you, a native speaker, prefer single quotes, I'm personally onboard!

But if there's any debate about it (let's see what the others think), I would very easily be convinced to use the Prettier default only because that way we can offload the decision to Prettier and avoid the debate. 😊

Reply:
Me too - I want single quotes but I would go with double of that's what most people want. 
 I'm very up for simple majority votes for things like this.  My overriding preference is meta: I want a clear process for deciding such things - the result is less important to me than how we get there :-)